python/paddle/fluid/tests/unittests/test_dist_base.py

```diff
@@ -187,6 +190,68 @@ def get_data():
             sys.stdout.buffer.write(pickle.dumps(out_losses))
 
 
+class TestParallelDyGraphRunnerBase(object):
+    def get_model(self):
+        raise NotImplementedError(
+            "get_model should be implemented by child classes.")
+
+    def run_one_loop(self, model, opt, data):
+        raise NotImplementedError(
+            "train_one_loop should be implemented by the child classes.")
+
+    def run_trainer(self, args):
+        seed = 90
+        device_id = int(os.getenv("FLAGS_selected_gpus", "0"))
```

Comment:
new flags？：FLAGS_selected_gpus

Reply:
Not a new flag, it's used to distinguish the GPU id.